x-pack/plugins/cases/public/client/helpers/can_use_cases.test.ts

```diff
@@ -6,140 +6,146 @@
  */
 
 import type { ApplicationStart } from '@kbn/core/public';
+import {
+  allCasesPermissions,
+  noCasesPermissions,
+  readCasesPermissions,
+  writeCasesPermissions,
+} from '../../common/mock';
 import { canUseCases } from './can_use_cases';
 
 type CasesCapabilities = Pick<
   ApplicationStart['capabilities'],
-  'securitySolutionCases' | 'observabilityCases'
+  'securitySolutionCases' | 'observabilityCases' | 'generalCases'
 >;
 
-const hasAll: CasesCapabilities = {
-  securitySolutionCases: {
-    crud_cases: true,
-    read_cases: true,
-  },
-  observabilityCases: {
-    crud_cases: true,
+interface Capabilities {
```

Comment:
Do we have a common type for that so we can import it?

Reply:
I removed this one in favor of the one in `test_providers.tsx`. I moved the `test_providers` type to our `common` directory of types.